remoteappmanager/file_config.py

```diff
@@ -77,18 +70,18 @@ def __init__(self, *args, **kwargs):
             self.docker_host = "unix://var/run/docker.sock"
 
         self.tls_verify = (env.get("DOCKER_TLS_VERIFY", "") != "")
```

Comment:
Not really strange, it's quite common to test for definedness, and docker is not the only one doing that. Putting 1 is just a convention. It could be anything.